s4e-web/cypress/page-objects/modal/general-modal.po.ts

```diff
@@ -3,7 +3,7 @@ import { Core } from './../core.po';
 export class GeneralModal extends Core {
   static pageObject = {
     getCloseBtnClass: '[data-e2e="close-btn"]',
-    getCloseBtn: () => cy.get(GeneralModal.pageObject.getCloseBtnClass),
+    getCloseBtn: () => cy.get(GeneralModal.pageObject.getCloseBtnClass), //X
```

Comment:
Please replace all names of keys in dict to more meaningful instead of writing comments.